src/consensus/replay_tower.zig

```diff
@@ -1579,6 +1613,622 @@ fn optimisticallyBypassVoteStakeThresholdCheck(
     return false;
 }
 
+pub fn collectVoteLockouts(
+    allocator: std.mem.Allocator,
+    logger: Logger,
```

Comment:
I think we should probably stop using unscoped loggers since it's so easy to convert the loggers now

```suggestion
    logger: ScopedLogger("replay_tower"),
```

Reply:
[7530482](https://github.com/Syndica/sig/pull/798/commits/75304829db9c578dad873d45f45bbbb7a1ee92c5)